src/main/java/graphql/introspection/Introspection.java

```diff
@@ -90,7 +90,7 @@ public enum TypeKind {
     public static final GraphQLEnumType __TypeKind = GraphQLEnumType.newEnum()
             .name("__TypeKind")
             .description("An enum describing what kind of type a given __Type is")
-            .value("SCALAR", TypeKind.SCALAR, "Indicates this type is a scalar. 'specifiedByUrl' is a valid field")
+            .value("SCALAR", TypeKind.SCALAR, "Indicates this type is a scalar. 'specifiedByURL' is a valid field")
```

Comment:
```
export interface IntrospectionScalarType {
  readonly kind: 'SCALAR';
  readonly name: string;
  readonly description?: Maybe<string>;
  readonly specifiedByURL?: Maybe<string>;
}
```


So it seems that graphql-js chose the name `specifiedByURL` and we chose `specifiedByUrl`

(Which is ironical because @andimarek drove this change in spec terms)

I think we should align to the reference implementation in this case - since it drives tooling behavior

But for backwards compat - we should "add" a new field and not change its name

Reply:
will https://github.com/graphql-java/graphql-java/pull/2993/commits/adafa6a3d73228a92619ef9861b0942d0f2d57dd#diff-eba216571550152c40136483beff9a41311df7b7681264e65d8d0e7c866cff7eR403 suffice? I could revert or add new tests for it as well.